lib/smart_answer/calculators/marriage_abroad_data_query.rb

```diff
@@ -152,13 +152,22 @@ def ss_unknown_no_embassies?(country_slug)
     def os_consular_cni_in_nearby_country?(country_slug)
       OS_CONSULAR_CNI_IN_NEARBY_COUNTRY.include?(country_slug)
     end
-    
+
+    def countries_with_18_outcomes
+      countries = marriage_data.fetch(:countries_with_18_outcomes)
+      valid_outcomes_country_data_structure?(countries) ? countries : []
+    end
+
     def marriage_data
       @marriage_data ||= YAML.load_file(path_to_data_file).with_indifferent_access
     end
 
   private
 
+    def valid_outcomes_country_data_structure?(countries)
+      countries.is_a?(Array) && countries.any? { |country| country.is_a?(String) }
```

Comment:
I think this wants to be `countries.all?` rather than `countries.any?`, as I think you want to say, "its an array, and all elements are strings".